docs/source/datasets.rst

```diff
@@ -2,43 +2,49 @@ Datasets
 ========
 
 To perform an systematic review, ASReview LAB requires a dataset representing
-all documents or publications obtained in a systematic search. To create such
-a dataset for a systematic review, typically an online library search is
+all records (e.g., abstracs of scientific papers) obtained in a systematic search. To create such
+a dataset for a systematic review, typically an `online library search <https://asreview.nl/the-importance-of-abstracts/>` is
 performed for all studies related to a particular topic.
 
-Each entry in the dataset should hold metadata on a paper.  Mandatory metadata
-are ``title`` and ``abstract``. Other metadata such as ``id``, ``author``,
-``date``, and ``keywords`` are optional. When using ASReview in simulation
-mode, an additional binary variable to indicate labeling decisions is
-required, called ``included`` or ``label_included``.  All entries in the
-dataset need to be screened, with the goal to select all relevant papers.
-
-For testing and demonstrating ASReview, the software offers
+It is possible to use
+`your own dataset <#using-your-own-data>`__ with unlabelled records, 
+partly labelled records (where the labelled records are used for training a model), 
+or fully labelled records (used for the Simulation mode).
+For testing and demonstrating ASReview (used for the Exploration mode), the software offers
 `three built-in datasets <#built-in-demonstration-data-sets>`__
-It is also possible to perform an automated systematic review on
-`your own dataset <#using-your-own-data>`__.
-
 
 Data format
 -----------
 
 To carry out an automated systematic review on your own dataset, your data file needs
 to adhere to a certain format. ASReview accepts the following formats:
 
- - `Research Information Systems (RIS) <https://en.wikipedia.org/wiki/RIS_(file_format)>`_.
+ - **RIS-files**.`Research Information Systems (RIS) <https://en.wikipedia.org/wiki/RIS_(file_format)>`_.
```

Comment:
Shall we replace "Research Information Systems (RIS)" with "RIS-files"? Makes no sense to most users.

Reply:
It was added by @SagevdBrand , so I leave it up to her to make this decision

Reply:
I agree!